test/test_baseparser.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: false
+
+require 'rexml/parsers/baseparser'
+
+module REXMLTests
+  class BaseParserTester < Test::Unit::TestCase
+    include REXML
+
+    N_ELEMENTS = 100
+    N_STRING = 'a' * 50000
```

Comment:
This name isn't good because `N_` means "the number of ".

Reply:
Oh, yes, indeed...
I understand.